frontend/beCompliant/src/components/Table.tsx

```diff
@@ -40,6 +41,28 @@ export function TableComponent({ data, fields }: TableComponentProps) {
     ),
   }));
 
+  columns.push({
+    header: ({ column }) => {
+      return <DataTableHeader column={column} header={'Når'} />;
+    },
+    id: 'Når',
+    accessorFn: (row) => row.fields['updated'] ?? '',
+    cell: ({ cell, getValue, row }: CellContext<any, any>) => (
+      <DataTableCell cell={cell}>
+        <TableCell
+          row={row}
+          value={getValue() ? formatDateTime(getValue()) : 'Ikke updatert'}
```

Comment:
```suggestion
          value={getValue() ? formatDateTime(getValue()) : 'Ikke oppdatert'}
```
vi har vel ikke diskutert om vi lander på bokmål/svensk/nynorsk 😂